PROCESS.md

```diff
@@ -0,0 +1,163 @@
+
+**Promotion Checklist:**
+[] There is general consensus that the feature has been proven to be a good
+  addition to JSON Schema and is unlikely to change.
+[] We see the feature being used successfully in the wild and not generating a
+  lot of community support issues that could indicate a problem.
+
+### STAGE-2
+`STAGE-2` features are in the last stages of becoming stable. Changes are
+unlikely, but backward incompatible changes are still allowed in extreme cases.
+Users can be reasonably sure that the feature won't change, but can't expect all
+implementations to support the feature yet.
+
+`STAGE-2` is a staging area for features that are ready to be declared stable,
+but are waiting for the next release or don't yet have enough implementations.
+When we see the vast majority of actively maintained implementations support
+this feature, it will be promoted to stable in the next release. Implementers
+SHOULD support `STAGE-2` features so they aren't out-of-date when the next
+release happens.
+
+Being in `STAGE-2` isn't a guarantee that a feature will be declared stable in
+the next release. If we don't see that the vast majority of actively maintained
+implementation have implemented the feature, it may stay in `STAGE-2` another
+year to allow implementations to catch up.
+
+`STAGE-2` features can only be promoted to stable as part of a release.
+
+**Promotion Checklist:**
+[] The vast majority of actively maintained implementations support the feature
+
+### Stable
+Any part of the specification that doesn't have stability flag or has a year
+stability flag such as `2024` is considered stable. This flag indicates the year
+a feature was declared stable. Any changes to stable features MUST be backward
+compatible and not introduce problems for forward compatibility. Stable features
+are never removed, but may be deprecated.
+
+### Deprecated
+Any feature in the specification that is flagged in the form `DEPRECATED-{YEAR}`
+is considered deprecated as of the year specified. Implementations SHOULD still
+support these features to maintain backward compatibility. Deprecated features
+will never be removed from the spec, but schema authors SHOULD migrate away from
+using them as implementations may begin dropping support for these features at
+some point.
+
+A feature can only be deprecated as part of a release.
+
+## Release Process
+Releases occur on Jan 1 every year. Each release is identified with the
+four-digit year of when the release occurred. Releases never introduce anything
+new. The only change should be updating `STAGE-2` flags to `{YEAR}` flags for
+the features that are promoted to stable that year, or adding adding deprecation
+flags for any features that are deprecated that year.
+
+Implementations that express support for a particular release MUST support all
+features that are stable as of that release.
+
+A snapshot of the spec will be taken of the stable parts of the spec and made
+available as a PDF on the website at.
```

Comment:
```suggestion
available as a PDF on the website at <placeholder>.
```

... if you intend to fill the blank later.

Reply:
Oops. Yeah, I had a placeholder originally and decided to remove it. I forgot to remove the "at" as well.